pkg/taxonomy/ext.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
What is this and how is it related to the tool?

Reply:
will do a cleanup